### PR TITLE
[opt](invert index) use lowercase by default

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — DISMISSED** [review]

File: be/src/olap/inverted_index_parser.cpp [hunk 1]
```diff
@@ -79,7 +79,7 @@ std::string get_parser_phrase_support_string_from_properties(
     if (properties.find(INVERTED_INDEX_PARSER_PHRASE_SUPPORT_KEY) != properties.end()) {
         return properties.at(INVERTED_INDEX_PARSER_PHRASE_SUPPORT_KEY);
     } else {
-        return INVERTED_INDEX_PARSER_PHRASE_SUPPORT_NO;
+        return INVERTED_INDEX_PARSER_FALSE;
```
Comment (airborne12) [inline be/src/olap/inverted_index_parser.cpp:82]: do not understand here

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**